pyathena/sqlalchemy_athena.py

```diff
@@ -289,6 +311,14 @@ class AthenaDialect(DefaultDialect):
         r"(((Database|Namespace)\ (?P<schema>.+))|(Table\ (?P<table>.+)))\ not\ found\."
     )
     _pattern_column_type = re.compile(r"^([a-zA-Z]+)($|\(.+\)$)")
+    construct_arguments: List[Tuple[DialectKWArgs, Mapping[str, Any]]] = [
```

Comment:
Add the following immediately below postfetch_lastrowid.
```
    construct_arguments = [
        (
            schema.Table,
            {"location": None, "compression": None},
        ),
    ]
```
Type annotation is not required for now. You can use `make fmt` to make code fomat.